x-pack/platform/plugins/shared/onechat/public/services/types.ts

```diff
@@ -6,12 +6,14 @@
  */
 
 import type { AgentService } from './agents';
+import { AgentProfilesService } from './agents/profiles';
```

Comment:
NIT: `import type` (if you have 2 or 3 hours I can explain why it matters on the browser-side, just ask)